### ISSUE TITLE
After installing WebKnossos on the server, I cannot log in

### ISSUE THREAD

**Issue by GanLab123** [scalableminds/webknossos#7484]
## Context

Why is it that after installing WebKnossos on the server following https://docs.webknossos.org/webknossos/installation.html, users can register but encounter 'Email or Password is wrong' when trying to log in? I didn't input the wrong password.
## Expected Behavior

Successfully logged in
## Current Behavior

During login, it prompts 'Email or Password is wrong'.
## Steps to Reproduce the bug

1.register
2.login

## Your Environment for bug

- Browser name and version: Microsoft Edge 119.0.2151.97
- Operating System and version: Windows 11
- Version of WEBKNOSSOS (Release or Commit):23.11.0 Release

**Comment by fm3** [comment]
Hi and thanks for your report!

Did this also happen for the very first created user (admin account)?

If it only happened for subsequently registered users, it might be that they need to be activated first. This is something that the admin account can do on the users page of webknossos. Compare the “Activating New Users” section at https://docs.webknossos.org/webknossos/users.html

Hope this helps!

**Comment by 913024733** [comment]
yes, I can't log in with the first created user.

**Comment by 913024733** [comment]
The content displayed in the logs.
![image](https://github.com/scalableminds/webknossos/assets/83834702/6e07ef08-5a38-4aaf-9308-dd3070575d80)

**Comment by fm3** [comment]
That’s interesting :thinking: 
What release of webknossos are you running? I.e. in the command
`DOCKER_TAG=xx.yy.z PUBLIC_HOST=webknossos.example.com LETSENCRYPT_EMAIL=admin@example.com \
docker compose up webknossos nginx nginx-letsencrypt` from the linked installation guide, what version number did you insert for xx.yy.z? also, it looks like PUBLIC_HOST may not have been set?

**Comment by 913024733** [comment]
Thank you for your response. I'm sorry. The earlier command I mentioned didn't specify a version number; I used the command [DOCKER_TAG=xx.yy.z PUBLIC_HOST=webknossos.example.com LETSENCRYPT_EMAIL=admin@example.com]. However, just now, I used the command [DOCKER_TAG=23.12.0 PUBLIC_HOST=webknossos.example.com LETSENCRYPT_EMAIL=admin@example.com], and encountered an issue while creating an admin account as shown in the image. Also, I'm not quite clear about the meaning of PUBLIC_HOST, and I haven't registered my own domain. Looking forward to your reply.
![image](https://github.com/scalableminds/webknossos/assets/83834702/b0b1ddd2-e5ab-480a-b9a3-b29eeb1563e3)

**Comment by 913024733** [comment]
Thank you for your help. I am currently working on deploying webknossos in a development environment, but I have encountered some issues. When uploading the dataset, I still receive the error message 'datastore server does not be available' (because other services are using port 9000, I have changed all instances of '9000' in the code to '19000'). I am still facing the same problem as I did when deploying it on Docker previously. Looking forward to your response.
![image](https://github.com/scalableminds/webknossos/assets/83834702/479304f6-1e44-47bb-b187-b264e8a15a9c)
![image](https://github.com/scalableminds/webknossos/assets/83834702/897dc0d0-87e2-4164-8b25-f1c1ca018d53)
